client/http/api.go

```diff
@@ -38,6 +38,7 @@ const (
 	store                     = "/pd/api/v1/store"
 	Stores                    = "/pd/api/v1/stores"
 	StatsRegion               = "/pd/api/v1/stats/region"
+	DistributionRegion        = "/pd/api/v1/distributions/region"
```

Comment:
How about "/pd/api/v1/regions/distribution"?

Reply:
keep the style with the "/pd/api/v1/stats/region"